libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -1317,7 +1317,8 @@ flattenPred pIn = go (freeVarNames pIn) [pIn]
 
 computeDependencies :: Pred fn -> DependGraph fn
 computeDependencies = \case
-  x@(Exists _ (_v :-> Block [Reifies (V a) _b _, _c])) -> deleteNode (Name a) (computeDependencies x)
+  -- @Tim this will inevitably loop, what's this supposed to do??
+  -- x@(Exists _ (_v :-> Block [Reifies (V a) _b _, _c])) -> deleteNode (Name a) (computeDependencies x)
```

Comment:
Oh, I forgot about this.  This was a failed experiment, that I didn't back out.
My apologies.